client/src/www/app/main.cordova.ts

```diff
@@ -71,6 +74,22 @@ class CordovaErrorReporter extends SentryErrorReporter {
   }
 }
 
+class CordovaMethodChannel implements MethodChannel {
+  async invokeMethod(methodName: string, params: string): Promise<string> {
+    switch (methodName) {
+      case 'FetchResource':
+        // TODO(fortuna): wire generic calls in the Cordova plugin.
+        return pluginExecWithErrorCode<string>('fetchResource', params);
```

Comment:
Should we `await` this call as well (or remove the `async` above and `await` below)?

Reply:
I added the await.